content/docs/1.5.5/volumes-and-nodes/pvc-ownership-and-permission.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Longhorn PVC ownership and permission
+weight: 1
+---
+
+In Kubernetes, there are 2 types of volume modes for PVC: Filesystem and Block ([link](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#volume-mode)).
+When a pod defines the security context and requests a Longhorn PVC, Kubernetes will handle the ownership and permission modification for the PVC differently for each type of volume mode.
+
+### Longhorn PVC with Filesystem volume mode
+
+Because Longhorn CSI driver `csiDriver.spec.fsGroupPolicy` sets to `ReadWriteOnceWithFSType`, Kubelet attempts to change the ownership and permission for Longhorn PVC in the following flow:
+1. If `pod.spec.securityContext.fsGroup` is non-empty, continue to the next step.
+   Otherwise, Kubelet doesn't attempt to change the ownership and permission for the volume.
+1. If the PV's `fsType` is defined and the PVC's `accessModes` list contains `ReadWriteOnly`, continue to the next step.
+   Otherwise, Kubelet doesn't attempt to change the ownership and permission for the volume.
+1. If the `pod.spec.securityContext.fsGroupChangePolicy` is set to `always` or empty:
+  1. Make sure that all processes of the containers inside the pod are part of the supplementary group id `pod.spec.securityContext.fsGroup`
+  1. Any new files created in the volume will be in group id `pod.spec.securityContext.fsGroup`
+  1. Kubelet recursively changes permission and ownership of the volume to have the same group id as `pod.spec.securityContext.fsGroup` everytime when the volume is mounted
+1. If the `pod.spec.securityContext.fsGroupChangePolicy` is set to `OnRootMismatch`:
+  1. If the root of the volume already has the correct permissions (i.e., belongs to the group id as `pod.spec.securityContext.fsGroup`) , the recursive permission and ownership change will be skipped.
+  1. Otherwise, Kubelet recursively changes permission and ownership of the volume to have the same group id as `pod.spec.securityContext.fsGroup`
+
+More references at:
+* https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#configure-volume-permission-and-ownership-change-policy-for-pods
+* https://github.com/longhorn/longhorn/issues/2131#issuecomment-778897129
```

Comment:
```suggestion
For more information, see [Configure volume permission and ownership change policy for Pods](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#configure-volume-permission-and-ownership-change-policy-for-pods) and [Issue #2131](https://github.com/longhorn/longhorn/issues/2131#issuecomment-778897129).
```

Reply:
Thank you. I incorporated part of your suggestion